Build currently fails due to an unknown identifier (IsWinXP2003)

This was originally noticed in the CI build for the WUA changes, but I just bumped into it while continuing to work on #263

`Utils.cpp

d:\a\legacyupdate\legacyupdate\legacyupdate\utils.cpp(23): error C3861: 'IsWinXP2003': identifier not found [D:\a\LegacyUpdate\LegacyUpdate\LegacyUpdate\LegacyUpdate.vcxproj]`

This is present after commit [Handle special case for Storage Server 2003 product name (https://github.com/LegacyUpdate/LegacyUpdate/issues/243)](https://github.com/LegacyUpdate/LegacyUpdate/commit/52c46cd21b05e1cdba0ccb8a1846aa5f602f5e2d), and we can fix it by including VersionInfo.h 😃

--- a/LegacyUpdate/Utils.cpp
+++ b/LegacyUpdate/Utils.cpp
@@ -3,6 +3,7 @@
 #include <atlstr.h>
 #include "HResult.h"
 #include "WMI.h"
+#include "VersionInfo.h"
 
 #pragma comment(lib, "version.lib")
 #pragma comment(lib, "advapi32.lib")

